cfn-resources/cloud-backup-snapshot/cmd/resource/resource.go

```diff
@@ -46,11 +46,14 @@ func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler
 		return *modelValidation, nil
 	}
 
-	// Create MongoDb Atlas Client using keys
-	client, err := util.CreateMongoDBClient(*currentModel.ApiKeys.PublicKey, *currentModel.ApiKeys.PrivateKey)
```

Comment:
Any idea why apiKeys were available in the currentModel while SnapshotId had to be saved in contextMap?